README.md

```diff
@@ -112,9 +112,23 @@ file, as discussed above.
 
 [issue-22]: https://github.com/GoogleCloudPlatform/gcsfuse/issues/22
 
-## Other performance issues
+# Support
 
-If you notice otherwise unreasonable performance, please [file an
+gcsfuse is an Open Source software released under the [Apache license][LICENSE]
+distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND.
```

Comment:
Does this really need to be quoted verbatim, in all caps? Better to use natural language here. You're also missing an end quote.